packages/webcomponents/src/components/form/form-control-monetary.scss

```diff
@@ -0,0 +1,20 @@
+@import 'reboot';
+@import 'forms';
+
+:host {
+  @include host-base-styles;
+  display: block;
+}
+
+input[type='text'][disabled] {
+  color: var(--jfi-form-control-disabled-color);
+  background-color: var(--jfi-form-control-disabled-background-color);
+  opacity: 0.5;
```

Comment:
Is this consistent with what we're doing for other controls? I think we should let [bootstrap take care of the `disabled` styling](https://getbootstrap.com/docs/5.3/forms/form-control/#disabled) (but still allow the css variable overrides)